ui/src/components/review/ReviewRow.js

```diff
@@ -44,7 +44,9 @@ export default class ReviewRow extends React.Component {
         super(props);
         this.api = this.props.api;
         this.onReview = this.onReview.bind(this);
-        let selectedOption = 'extend';
+        let selectedOption = this.props.expiryOrReviewSettingIsSet
+            ? 'extend'
+            : 'no-action';
         this.state = {
```

Comment:
if expiry or review days set, then default option is extend, otherwise - no action